### PR TITLE
Chore Fixes (Diary / Topics / Article Page)

### PR DESCRIPTION
**Pull request by Drapegnik**
## read article in lang
Screen Shot 2019-10-10 at 22 19 37

## other
- fix personalities page crash
- refactor diary with hooks, fix next to the future bug
- change `<title />` on article page

### CODE REVIEW

**Comment by Drapegnik** [comment]
[Diary Fixes](https://trello.com/c/ZsQ7Z16J/408-diary-fixes)

**Comment by Drapegnik** [comment]
[Topic Page](https://trello.com/c/wp2cnZ7W/419-topic-page)

**Review by UladBohdan — DISMISSED** [review]

**Comment by UladBohdan** [comment]
[hover for cards on touch vs. desktop devices](https://trello.com/c/plM7kzAX/430-hover-for-cards-on-touch-vs-desktop-devices)

File: package.json [hunk 1]
```diff
@@ -128,40 +128,40 @@
...
     },
     {
       "path": ".next/static/**/tag.js",
-      "maxSize": "4 kB"
+      "maxSize": "2 kB"
     },
     {
       "path": ".next/static/**/about.js",
-      "maxSize": "800 B"
+      "maxSize": "8 kB"
```
Comment (UladBohdan) [inline package.json:156]: why did `/about` grow up?

Comment (UladBohdan) [reply package.json:156]: unlike all other assets

**Review by UladBohdan — APPROVED** [review]

**Pull request merged**